src/librustc_llvm/build.rs

```diff
@@ -263,7 +263,12 @@ fn main() {
         } else if cxxflags.contains("stdlib=libc++") {
             println!("cargo:rustc-link-lib=c++");
         } else {
-            println!("cargo:rustc-link-lib={}", stdcppname);
+            if target.contains("powerpc64-unknown-freebsd") {
+                println!("cargo:rustc-link-search=native=/usr/local/lib/gcc6");
```

Comment:
Does `llvm-config` not print this path out? If I remember my freebsd correctly, this would mean that `gcc` would be required to build rustc then. Which is not that big of a deal, but still something to improve on.

Reply:
FreeBSD on powerpc64 ships an old version of gcc (4.2.1) in the 'base' system, since it can't build llvm I had to use gcc7 from our 'ports tree'. For whatever reason the linker tried to link 'something' against libstdc++ from gcc-4.2.1 instead of libstdc++ from gcc7 and the link failed (I was not able to figure out how to pass -Wl,-rpath=/usr/local/lib/gcc7). Linking to a static version of libstdc++ made the problem go away.
This is just an ugly hack to fix a FreeBSD problem and should probably stays as a patch file in our 'ports tree'